x-pack/test/security_solution_endpoint_api_int/services/roles_users.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Role } from '@kbn/security-plugin/common';
+
+import { getT1Analyst } from '@kbn/security-solution-plugin/scripts/endpoint/common/roles_users/t1_analyst';
+import { getT2Analyst } from '@kbn/security-solution-plugin/scripts/endpoint/common/roles_users/t2_analyst';
+import { getHunter } from '@kbn/security-solution-plugin/scripts/endpoint/common/roles_users/hunter';
+import { getThreadIntelligenceAnalyst } from '@kbn/security-solution-plugin/scripts/endpoint/common/roles_users/thread_intelligence_analyst';
+import { getSocManager } from '@kbn/security-solution-plugin/scripts/endpoint/common/roles_users/soc_manager';
+import { getPlatformEngineer } from '@kbn/security-solution-plugin/scripts/endpoint/common/roles_users/platform_engineer';
+import { getEndpointOperationsAnalyst } from '@kbn/security-solution-plugin/scripts/endpoint/common/roles_users/endpoint_operations_analyst';
+import { getEndpointSecurityPolicyManager } from '@kbn/security-solution-plugin/scripts/endpoint/common/roles_users/endpoint_security_policy_manager';
+
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export enum ROLE {
+  t1_analyst = 't1Analyst',
+  t2_analyst = 't2Analyst',
+  analyst_hunter = 'hunter',
+  thread_intelligence_analyst = 'threadIntelligenceAnalyst',
+  detections_engineer = 'detectionsEngineer',
+  soc_manager = 'socManager',
+  platform_engineer = 'platformEngineer',
+  endpoint_operations_analyst = 'endpointOperationsAnalyst',
+  endpoint_security_policy_manager = 'endpointSecurityPolicyManager',
```

Comment:
Why are these `enum` members written in snake_case?

Reply:
No specific reason, just in order to differentiate the key of the value